btcec/ciphering.go

```diff
@@ -22,14 +23,19 @@ func GenerateSharedSecret(privkey *PrivateKey, pubkey *PublicKey) []byte {
 
 // Encrypt encrypts data for the target public key using AES-128-GCM
```

Comment:
format: Comments should be full sentences: https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#code-documentation-and-commenting